src/components/charts/galaxySelector/galaxySelectorUtilities.js

```diff
@@ -0,0 +1,20 @@
+import findIndex from 'lodash/findIndex';
+
+export const getFluid = image => {
+  const {
+    childImageSharp: { fluid },
+  } = image;
+  return fluid;
+};
+
+export const parseName = originalName => {
+  return originalName.split('_sci.')[0];
+};
+
+export const getNameFromImage = image => image.id;
+
+export const getActiveIndex = (images, activeId) => {
+  return findIndex(images, image => {
+    return getNameFromImage(image) === activeId;
+  });
+};
```

Comment:
please remove any unused functions from the files you copied in

Reply:
I believe the only function that I'm not using from this file any longer is the getFluid function. I have removed it and will be pushing that change up with my most recent commit.